cs.html

```diff
@@ -961,689 +1090,685 @@ <h2>Constructors</h2>
-               underlying data transport.</li>
-               <li>if <var>data</var>.finish is set to <code>true</code>, run the
-               following:</li>
-               <ol>
-                 <li>Queue a STREAM frame with the FIN bit set.</li>
-                 <li>Set <var>stream</var>'s <a>[[\Writable]]</a> slot to
-                 <code>false</code>.</li>
-                 <li>Let <var>transport</var> be the <code><a>QuicTransportBase</a></code>,
-                 which the <var>stream</var> was created from.</li>
-                 <li>Remove the <var>stream</var> from the <var>transport</var>'s
-                 <a>[[\QuicTransportWritableStreams]]</a> internal slot.</li>
-               </ol>
-               <div class="note">The actual transmission of data occurs in
-               parallel. If sending data leads to a QUIC-level error, the
-               application will be notified asynchronously through the
-               <code><a>QuicTransportBase</a></code>'s <code><a>onerror</a></code>
-               EventHandler.</div>
-              </ol>
-              <table class="parameters">
-                <tbody>
-                  <tr>
-                    <th>Parameter</th>
-                    <th>Type</th>
-                    <th>Nullable</th>
-                    <th>Optional</th>
-                    <th>Description</th>
+                    <td class="prmDesc">The host to connect to.</td>
                   </tr>
                   <tr>
-                    <td class="prmName">data</td>
-                    <td class="prmType"><code>QuicStreamWriteParameters</code></td>
+                    <td class="prmName">port</td>
+                    <td class="prmType"><code><a>unsigned short</a></code></td>
                     <td class="prmNullFalse"><span role="img" aria-label=
                     "False">&#10008;</span></td>
                     <td class="prmOptFalse"><span role="img" aria-label=
                     "False">&#10008;</span></td>
-                    <td class="prmDesc"></td>
+                    <td class="prmDesc">The port to connect to.</td>
                   </tr>
                 </tbody>
               </table>
-              <div>
-                <em>Return type:</em> <code>void</code>
-              </div>
             </dd>
-            <dt><dfn><code>abortWriting</code></dfn></dt>
-            <dd>
-              <p>A hard shutdown of the <code><a>QuicWritableStream</a></code>. It may be called
-              regardless of whether the <code><a>QuicWritableStream</a></code>
-              was created by the local or remote peer. When the <code>abortWriting()</code>
-              method is called, the user agent <em class="rfc2119" title="MUST">MUST</em>
-              run the following steps:</p>
+          </dl>
+        </section>
+      </div>
+    </section>
+  </section>
+
+  <section id="outgoing-stream*">
+    <h2>Interface Mixin <dfn>OutgoingStream</dfn></h2>
```

Comment:
Why the change from WritableStream to OutgoingStream?

Reply:
and with ReadableStream as well.

Reply:
To avoid name conflicts with WHATWG streams (which we may use in the future).

Reply:
Got it.